.travis.yml

```diff
@@ -2,18 +2,15 @@ sudo: true
 dist: trusty
 language: node_js
 node_js:
-- '5'
+- '6'
 before_script:
-- npm install bower
+- npm install -g bower web-component-tester
 - bower install
-script: wct --plugin sauce
-cache:
-  directories:
-  - node_modules
+script: xvfb-run wct
```

Comment:
Sauce is broken for us, if you haven't noticed. Don't know if they changed their plans or what, but always says we're out of minutes